<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>731</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_join">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>290</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_id">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>아이디를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_pwd">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>180</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>비밀번호를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_page">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디/비밀번호 찾기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_id">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_id">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>아이디를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;회원가입&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_pwd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_submit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>360</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pwd_chk">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호확인</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancel">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>360</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_pwd">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>비밀번호를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_pwd_chk">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>비밀번호를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_tel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>'-'를 제외한 숫자로만 입력해 주세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>연락처</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLineEdit" name="search_id">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>250</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>아이디를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_tel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>'-'를 제외한 숫자로만 입력해 주세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancel2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>330</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;아이디 비밀번호 찾기&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>330</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tel2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>연락처</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_id2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="search_gubun1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디 찾기</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="search_gubun2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>150</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호 찾기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>로그인 성공</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>201</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Downloads/pngwing.com(1).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QListWidget" name="list_widget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>661</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_list2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="insert_spin">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>330</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>290</y>
       <width>150</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_insert">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>330</y>
       <width>150</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>INSERT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_print">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>150</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>PRINT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_print_multi">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>150</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>PRINT_MULTI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>380</y>
       <width>150</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>380</y>
       <width>150</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>REMOVE</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>331</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pte">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>Print Text Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cte">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>Clear Text Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sf">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>180</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>Set Font</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sfi">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>240</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>Set Font Italic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sfcr">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>300</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(28, 113, 216);</string>
     </property>
     <property name="text">
      <string>Set Font Color Red</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="input_fs">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>400</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Z003</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>370</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Font Size</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
